doc/menu.md

```diff
@@ -0,0 +1,204 @@
+<!---
```

Comment:
Once we "fix" the links per above to work in GH, then persumably this menu _would_ work.  Until then, it makes sense to remove the file.  So, I'll go ahead and remove from this PR, then when we sort out the above, we can consider if it makes sense to put it back.